### PR TITLE
readme.md: Add JS testing and stable branch instructions

### PR DESCRIPTION
**Pull request by oskosk**
#### Changes proposed in this Pull Request:
- Adds reference to installation from git repo with mention to the `master-stable` branch.
- Adds instructions for running Admin Page unit tests. 

#### Testing instructions

1. View the new [README.md as updated by this branch](https://github.com/Automattic/jetpack/blob/add/testing-and-stable-instructions/readme.md).
1. Check the new 2nd level section **Installation from git repo**.
1. Check the new 4th level section **Unit-testing the JS Admin Page**.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,6 +14,17 @@ To install the Jetpack plugin on your site, [follow the instructions on this pag
 
 Do you need help installing Jetpack, or do you have questions about one of the Jetpack modules? You can [search through our documentation here](http://jetpack.com/support/). If you don't find the answers you're looking for, you can [send us an email](http://jetpack.com/contact-support/) or [start a new thread in the WordPress.org support forums](https://wordpress.org/support/plugin/jetpack#postform).
 
+## Installation from  git repo
+
+The `master-stable` branch of this repo contains an stable version with every JS file pre-built.
```
Comment (dereksmart) [inline readme.md]: JS && CSS

File: readme.md [hunk 2]
```diff
@@ -14,6 +14,17 @@ To install the Jetpack plugin on your site, [follow the instructions on this pag
...
+
+```
+$ cd wp-content/plugins
+$ git clone https://github.com/Automattic/jetpack
```
Comment (jeherve) [inline readme.md]: I'd recommend using `git@github.com:Automattic/jetpack.git` instead, so you don't need a password every time you want to push.

File: readme.md [hunk 3]
```diff
@@ -14,6 +14,17 @@ To install the Jetpack plugin on your site, [follow the instructions on this pag
 
 Do you need help installing Jetpack, or do you have questions about one of the Jetpack modules? You can [search through our documentation here](http://jetpack.com/support/). If you don't find the answers you're looking for, you can [send us an email](http://jetpack.com/contact-support/) or [start a new thread in the WordPress.org support forums](https://wordpress.org/support/plugin/jetpack#postform).
 
+## Installation from  git repo
+
+The `master-stable` branch of this repo contains an stable version with every JS file pre-built.
+
+```
+$ cd wp-content/plugins
```
Comment (jeherve) [inline readme.md]: Maybe say something like "From your Plugins directory", since it may be in a different location for some people.

**Comment by oskosk** [comment]
I've just applied every fix and recommendations by @jeherve 

* Mention "your plugins directory" instead of stating `wp-content/plugins` 
* Use SSH URL for the repo
* State that also CSS is built as well as JS code.

**Comment by dereksmart** [comment]
This LGTM, but do we want to change the reference to `master-stable` until it's accurate?  Right now the prebuilt lives on `jetpack-built`

**Comment by oskosk** [comment]
I've updated the branch reference to `jetpack-built`. Merging

**Pull request merged**